src/app/learn/get-started/basics/build-publish/index.html

```diff
@@ -0,0 +1,43 @@
+<stache
+  navOrder="7"
+  pageTitle="Build and Publish a SPA"
+  navTitle="Build and publish">
+
+  <stache-page-summary>
+    <p>
+      After you update the SKY UX CLI with the latest version, you can build and publish your SPA. 
+    </p>
+    <sky-alert alertType="info">
+      Blackbaud developers should create repos and provision applications through the SKY UX App Management Portal. Documentation on the SKY UX App Management Portal is available on <a href="https://host.nxt.blackbaud.com/engineering-system-docs/learn/spa/spa-app-portal">the Engineering System website</a>. This site is for Blackbaud internal use only and is not available to external developers.
+    </sky-alert>
+  </stache-page-summary>
+  
+  <ol>
+    <li>
+      <p>
+        From the command prompt, <stache-code>cd</stache-code> into the application's directory.
+      </p>
+    </li>
+    <li>
+      <p>
+        Enter <stache-code>skyux build</stache-code> to build the SPA into the <stache-code>dist/</stache-code> folder.
+      </p>
+    </li>
+    <li>
+      <p>
+        Copy the contents of the <stache-code>dist/</stache-code> folder into a directory such as <stache-code>my-spa/</stache-code>, where<stache-code>my-spa</stache-code> is the name of your SPA.
+      </p>
+    </li>
+    <li>
+      <p>
+        Store the <stache-code>my-spa/</stache-code> directory on a web server.
+      </p>
+    </li>
+    <li>
+      <p>
+        In a browser, enter <stache-code>https://example.com/my-spa</stache-code> to view your SPA.
+      </p>
+    </li>
+  </ol>
+  
```

Comment:
We should probably add a caveat about the need for redirecting all requests to the root `index.html` file.  The instructions are different for whatever host/server they chose so there's no one-answer for everyone, but calling it out might at least be importing.  Something like... "Since this produces a single page application that uses HTML5 push state for routing, you will need to configure your webserver / host to redirect all requests to your root `index.html` file." (or something more eloquent)?

Reply:
Added this as a note. Open to @blackbaud-johnly's feedback on the wording and callout.